app/storage/metadata_parser.py

```diff
@@ -55,10 +55,11 @@ class RunnerMetadataSchema(Schema, StripWhitespaceMixin):
     ru_ref = VALIDATORS['string'](validate=validate.Length(min=1))
     collection_exercise_sid = VALIDATORS['string'](validate=validate.Length(min=1))
     tx_id = VALIDATORS['uuid']()
-    case_id = VALIDATORS['uuid']()
+    questionnaire_id = VALIDATORS['string']()
```

Comment:
We should add length to these as currently, empty strings would be valid.

Reply:
I've changed these for now, but for case_id/questionnaire_id - I wonder if we should be validating them at all. Given that we don't use them for anything - we probably should just be passing them on?

Reply:
Possibly, guess it depends on how mandatory it actually is. Do we return a 403 if they don't provide us a questionnaire_id vs rm to preventing launching without questionnaire_id.

Reply:
questionnaire_id will cause a failure, case_id will not

Reply:
YaI think your change is fine to validate questionnaire_id/response_id since they are needed but not case_id